azure-pipelines.yml

```diff
@@ -36,23 +36,23 @@ jobs:
           PYTHON_ARCH: 'x64'
           BUILD_TYPE: 'normal'
           imageName: 'windows-2019'
-        Win-Python39-64bit-full-wheel:
-          PYTHON_VERSION: '3.9'
+        Win-Python310-64bit-full-wheel:
```

Comment:
lxml missing wheels are a Cython level issue for win64. We need to get wheel generation via cibuildwheels up by the 1st, so let's just add py3.11 windows checking at a later date (probably when lxml does its next release).